tst/working/quick/ProjSubfield2.tst

```diff
@@ -5,6 +5,6 @@ gap> x := PseudoRandom(GL(6,5^3));;
 gap> gens := List(gens,y->y^x);;
 gap> g := GroupWithGenerators(gens);;
 
-# Remark: The *6 comes from the fact that by blowing up we lose a factor
+# Remark: The *6 comes from the fact that by blowing up we lose a image
```

Comment:
```suggestion
# Remark: The *6 comes from the fact that by blowing up we lose a factor
```

Reply:
Huh, strange. I'm pretty sure I reverted that replacement. Maybe I messed up some git stuff? 🤔 Anyways, thanks again. :)